.github/workflows/Hello-CI.yml

```diff
@@ -73,3 +73,25 @@ jobs:
             --simlimit 60 --stat
             echo " Show simulation UART output build with GCC Release"
             cat ./out/Hello/CS300/Release/GCC_fvp_stdout.log
+

```

Comment:
While the changes function as intended, there is noticeable repetition in the instructions and steps for each toolchain, which are currently executed sequentially.
This setup has a drawback if one toolchain build fails, it halts the execution of subsequent steps.
I recommend consolidating the repeated instructions and configuring the jobs to run in parallel for each toolchain.

Reply:
Point taken. I leave it to you to decide whether @Evagan2018 would create a matrix build or you (@soumeh01)